crates/blockifier/src/execution/contract_class.rs

```diff
@@ -546,13 +548,28 @@ impl SierraContractClassV1 {
         Some(contract_entrypoint_to_entrypoint_selector(entrypoint))
     }
 
-    pub fn try_from_json_string(
-        raw_contract_class: &str,
-    ) -> Result<SierraContractClassV1, ProgramError> {
+    pub fn try_from_json_string(raw_contract_class: &str) -> Result<Self, ProgramError> {
+        fn compile(sierra_program: &SierraProgram) -> AotNativeExecutor {
+            let native_context = cairo_native::context::NativeContext::new();
```

Comment:
We are performing this operation once per new contract? How costly it is?

Reply:
Yes this happens once per new contract. I did not measure a drop in performance compared to using a global static with `sync::OnceLock` when processing a single block.